packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -361,6 +361,8 @@
 	"dataMapping.jsonHint": "<img src='/static/json-mapping-gif.gif'/> Drag a JSON key onto <b>{name}</b> to map data",
 	"dataMapping.mapKeyToField": "Map '{name}' to a field",
 	"dataMapping.mapAllKeysToField": "Map every '{name}' to a field",
+	"dataMapping.tableView.tableColumnsExceeded": "Some columns are hidden.",
+	"dataMapping.tableView.tableColumnsExceededTooltip": "Your data has more than {columnLimit} columns so some are hidden. Switch to <a data-action='switchToJsonView{paneType}'>JSON view</a> to see all data.",
```

Comment:
would not using `<i18n> ` cleaner approach than a global action?

Reply:
thanks for the tip, applied